cluster/juju/layers/kubernetes-master/Makefile

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/make
+
+all: lint unit_test
+
+
+.PHONY: clean
+clean:
+	@rm -rf .tox
+
+.PHONY: apt_prereqs
+apt_prereqs:
+	@# Need tox, but don't install the apt version unless we have to (don't want to conflict with pip)
+	@which tox >/dev/null || (sudo pip install tox)
+	@sudo apt-get install -y python-pip python-flake8 python3-flake8 flake8
```

Comment:
is python pip required here? we're also calling pip install tox above with the possibility that pip isn't installed.